app/controllers/people_controller.rb

```diff
@@ -110,7 +110,7 @@ def create
     @person = Person.new(person_params)
     cookies[:status] = person_params[:status]
     respond_to do |format|
-      if @person.save
+      if @person.save && @person.application_and_start_validation
```

Comment:
I would like to see the check as a validation on the model. That's where Rails convention wants validation.
While not the exact same situation, you may want to take a look at
https://github.com/ReadyResponder/ReadyResponder/blob/development/app/models/person.rb#L37
and
https://github.com/ReadyResponder/ReadyResponder/blob/development/config/initializers/validators.rb#L1

Thanks for your help.